src/ceremony_and_credits.c

```diff
@@ -1752,17 +1560,17 @@ struct struct_80284AE8 {
     u8 unk1C;
 };
 
-void play_cutscene(CinematicCamera *camera) {
+void play_cutscene(struct struct_D_802876D0 *camera) {
     s32 pad[3];
     s16 cutsceneDuration;
 
-#define CUTSCENE(id, cutscene)                                                                            \
-    case id:                                                                                              \
-        cutsceneDuration = cutscene[sCutsceneShot].duration;                                                  \
-        cutscene[sCutsceneShot].shot(camera);                                                                  \
+#define CUTSCENE(id, cutscene) \
+    case id: \
+        cutsceneDuration = cutscene[sCutsceneShot].duration; \
+        cutscene[sCutsceneShot].shot(camera); \
 
-    if (!camera->cutscene) { return; }
-    switch (camera->cutscene) {
```

Comment:
This naming is from sm64 and is likely accurate as the code is pretty similar.

However, this is probably something we can come back to later. Probably fine for now.

Reply:
If you want, I could rename `struct_D_802876D0 ` to `CinematicCamera` and carry the relevant member names too. Stuff like `pos`, `lookat`, and `cutscene`

Reply:
Yea, if stuff seems relevant for sure. (Don't have to label everything, just whatever few makes sense).

I'll probably go back through again and see if I can clean it up a bit more (if there's any remaining left over unused structs).

CinematicCamera is probably not the best name. I think it's suitable for now unless you can think of something better.